native/src/components/base/TextButton.tsx

```diff
@@ -25,12 +25,31 @@ type TextButtonProps = {
   text: string
   onPress: () => Promise<void> | void
   disabled?: boolean
+  accessible?: boolean
+  focusable?: boolean
```

Comment:
Please remove the `accessible` and `focusable` props from this component, as you do not use them and they are set by default for every `Pressable`, see the React Native docs: https://reactnative.dev/docs/accessibility#accessible